### PR TITLE
Option 1 and 2 are done.

### PR DESCRIPTION
**Pull request by EstebanBrito**

### CODE REVIEW

**Comment by EstebanBrito** [comment]
Just accept it

File: main.py [hunk 1]
```diff
@@ -1,36 +1,126 @@
...
 def touch(filename):
-    """Función para crear un archivo"""
+    """Función para crear un archivo con texto aleatorio dentro"""
     subprocess.call(["touch", filename])
     os.system('echo ' + lorem.paragraph() + ' > ' + filename)
 
+def getRandomName():
+    """Retorna un nombre aleatorio"""
+    num = random.randint(0, len(names)-1)
+    return names[num]
```
Comment (42B) [inline main.py]:
```python
return random.choice(names)  # ?
```

File: main.py [hunk 2]
```diff
@@ -1,36 +1,126 @@
...
+
+def getRandomFile(dirname="."):
+    """Retorna de forma aleatoria el nombre de un archivo dentro de dirname"""
+    files = [f for f in os.listdir(dirname) if os.path.isfile(f)]
+    if len(files) > 0:
+        num = random.randint(0, len(files)-1)
+        name = files[num]
+        return name
```
Comment (42B) [inline main.py]:
```python
return random.choice(files)  # ?
```

File: main.py [hunk 3]
```diff
@@ -1,36 +1,126 @@
...
+    biggest_size = 0
+    for f in files:
+        size = os.stat(dirname + "/" + f).st_size
+        if size > biggest_size:
+            biggest_size = size
+            biggest_file = f
+    return biggest_file, biggest_size
+
+
+def getSmalestFile(dirname):
```
Comment (42B) [inline main.py]: "Smalest" -> "Smallest"?

**Pull request merged**